instrumentation/helpers-sql-obfuscation/test/test_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'bundler/setup'
+Bundler.require(:default, :development, :test)
+
+SimpleCov.start
+SimpleCov.minimum_coverage 85
```

Comment:
@ericmustin - I added SimpleCov because it was present in the `base` test helper. However, it doesn't seem like it's working in either scenario. 

For the new helper gems, I'm getting `0 / 0 LOC (100%) covered.` 
`base` shows `5 / 5 LOC (100%) covered.`, with coverage only for the instrumentation test itself. 

At New Relic, we had to put a decent chunk of time in to get SimpleCov to work with our funky testing setup. Something similar may need to be done here. 

I'm leaning toward pulling SimpleCov out of the new helper gems for now since it's not working. wdyt?

Reply:
i dont think we are using SimpleCov in practice for anything, that's fine with me.